torchgeo/trainers/byol.py

```diff
@@ -417,40 +411,36 @@ def training_step(  # type: ignore[override]
         Returns:
             training loss
         """
+        batch, batch_idx = args
```

Comment:
Are you sure this works? It seems like it wouldn't from https://pytorch-lightning.readthedocs.io/en/stable/common/lightning_module.html#training-step.